drivers/hud_twenty_twenty_two_to_twenty_twenty_four/app/models/hud_twenty_twenty_two_to_twenty_twenty_four/project/update_project_type.rb

```diff
@@ -0,0 +1,25 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HudTwentyTwentyTwoToTwentyTwentyFour::Project
+  class UpdateProjectType
+    def process(row)
+      project_type = if row['ProjectType'].to_i == 1
+        if row['TrackingMethod'].to_i == 3
+          1
+        else
+          0
+        end
+      else
+        row['ProjectType']
+      end
```

Comment:
Nothing wrong with what you have, but maybe:
```suggestion
      project_type = if row['ProjectType'].to_i == 1 && row['TrackingMethod'].to_i == 3
          0
      else
        row['ProjectType']
      end
```

Reply:
I'm going to stick w/ mine... I think yours needed to be `row['TrackingMethod'].to_i == 0` which is fine, but if we both had to work through it to know if it is correct...